sdk/tools/azure-openrewrite/src/test/resources/migrationExamples/azure-core/sample1/v2/HttpRequestExample.java

```diff
@@ -0,0 +1,22 @@
+import io.clientcore.core.http.client.HttpClient;
+import io.clientcore.core.http.client.JdkHttpClientBuilder;
+import io.clientcore.core.http.models.HttpMethod;
+import io.clientcore.core.http.models.HttpRequest;
+import io.clientcore.core.http.models.Response;
+import java.io.IOException;
+
+public class HttpRequestExample {
+    public static void main(String... args) {
+        HttpClient client = new JdkHttpClientBuilder().build();
```

Comment:
This isn't right - we shouldn't be changing the customer code from using Netty to using JdkHttpClient